bodhi/server/services/releases.py

```diff
@@ -327,3 +346,55 @@ def save_release(request):
     request.db.flush()
 
     return r
+
+
+@critical_packages.put(schema=bodhi.server.schemas.UpdateCriticalPackages,
+                       renderer='json',
+                       error_handler=bodhi.server.services.errors.json_handler,
+                       validators=(colander_body_validator, validate_tags, validate_enums)
+                       )
+def add_packages(request):
+    """
+    Add a list of critical path packages to a specific release.
+
+    Args:
+        request (pyramid.request): The current request.
+    Returns:
+        dict: The dictionary {'status': 'success'}.
+    """
+    data = request.validated
+    pkg_list = data.pop('packages', None)
```

Comment:
We should use a validator to ensure that the pkg_list is already Package objects and not package names.